src/10_functions.py

```diff
@@ -2,6 +2,12 @@
 
 # YOUR CODE HERE
 
+def is_even(number):
+  if number % 2 == 0:
+    return True
+  else:
+    return False
+
```

Comment:
Can be simplified to this:
def is_even(n):
  return n % 2 == 0

Reply:
Good call.